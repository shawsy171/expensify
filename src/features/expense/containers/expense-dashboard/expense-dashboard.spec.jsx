import React from 'react';
import { shallow } from 'enzyme';

import ExpenseDashBoard from './expense-dashboard';

describe('ExpenseDashBoard Component', () => {
  test('should render correctly', () => {
    const wrapper = shallow(<ExpenseDashBoard />);
    expect(wrapper).toMatchSnapshot();
  });
});
