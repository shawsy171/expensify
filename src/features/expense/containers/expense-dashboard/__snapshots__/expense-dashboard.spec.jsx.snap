// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExpenseDashBoard Component should render correctly 1`] = `
<div>
  <p>
     this is the Expense DashBoard 
  </p>
  <Connect(ExpenseSummary) />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
